libparsec/crates/platform_mountpoint/src/lib.rs

```diff
@@ -0,0 +1,103 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+#![doc = include_str!("../README.md")]
+
+mod error;
+mod memfs;
```

Comment:
hide it behind a feature ? or move it into the tests folder ?

Reply:
`MountpointManager` is used by `windows`, no other `MountpointManager` is implemented to replace it